persistence/src/main/java/edu/unc/lib/dl/persist/services/transfer/StreamToFSTransferClient.java

```diff
@@ -98,7 +98,7 @@ protected URI writeStream(PID binPid, InputStream sourceStream, boolean allowOve
             }
 
             // Move temp file into final location
-            Files.move(newFilePath, destPath, REPLACE_EXISTING);
```

Comment:
we still want to use move for renaming the temp/old files since it is within the same storage location, its just for source files (which doesn't exist for this client) that we want to switch to copy